cookbook/form/data_transformers.rst

```diff
@@ -434,23 +397,69 @@ Now, whenever you need to use your special ``issue_selector`` field type,
 it's quite easy::
 
     // src/AppBundle/Form/TaskType.php
-    namespace AppBundle\Form;
-
-    use Symfony\Component\Form\AbstractType;
-    use Symfony\Component\Form\FormBuilderInterface;
+    use AppBundle\Form\DataTransformer\IssueToNumberTransformer;
 
     class TaskType extends AbstractType
     {
         public function buildForm(FormBuilderInterface $builder, array $options)
         {
             $builder
-                ->add('task')
-                ->add('dueDate', null, array('widget' => 'single_text'))
+                ->add('description', 'textarea')
                 ->add('issue', 'issue_selector');
         }
 
-        public function getName()
-        {
-            return 'task';
-        }
+        // ...
     }
+
+About Model and View Transformers
+---------------------------------
+
+In the above example, the transformer was used as a "model" transformer.
+In fact, there are two different types of transformers and three different
+types of underlying data.
+
+.. image:: /images/cookbook/form/DataTransformersTypes.png
+   :align: center
+
+In any form, the three different types of data are:
+
+1) **Model data** - This is the data in the format used in your application
+   (e.g. an ``Issue`` object). If you call ``Form::getData`` or ``Form::setData``,
+   you're dealing with the "model" data.
+
+2) **Norm Data** - This is a normalized version of your data, and is commonly
```

Comment:
I don't think it's a serial (no list), but I agree it should be removed (no reason to "pause" right there)